recipes/gpaw/conda_build_config.yaml

```diff
@@ -0,0 +1,12 @@
+python:
```

Comment:
I might be missing something but could `python` be removed from here and `skip: True  # [py<36]` be added to the `build` section of `meta.yaml` instead?